ee/tools/puppet/fleetdm/examples/multiple-teams.pp

```diff
@@ -0,0 +1,10 @@
+node default {
+  fleetdm::profile { 'com.apple.universalaccess':
+    template => 'xml template',
+    group    => 'workstations',
+  }
+
+  fleetdm::profile { 'com.apple.homescreenlayout':
+    template => 'xml template',
```

Comment:
The `group` directive is not required? So in this case this profile would not add a label to the generated team name? (gathering info for the work on my ticket at the same time :D )

Reply:
It's not required but has a default value of `"default"`. We could also make it required!

Reply:
Either way sounds good to me. I'll have to dedupe those labels anyway, and possibly add a random part to avoid any  conflict (as team name is unique).

Reply:
@mna damnit, I missed speccing that (will update now) but the random part should be a timestamp in the form `2022-05-12:01:03:12`.

For example: `workstations - canary (2022-05-12:01:03:12)`

Reply:
:+1: cool, no worry I was just starting to implement team creation and hadn't yet added a random part! I would dedupe and sort the "group" labels though, unless there are explicit requirements there (e.g. your example would become `canary - workstations (...)`).

Reply:
that sounds good 👍